### ISSUE TITLE
Preformace terrible after upgrade

### ISSUE THREAD

**Issue by asdf23** [zeit/hyper#6060]
- [X] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10
- **Hyper.app version**: 3.1.4
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: No modifications
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: Vanilla 

## Issue
The performance is terrible after upgrading. I notice this especially when I open two tabs and then on the second tab navigate through the history (press up a few times). Or when pressing Ctrl+Left to jump to the previous word in a line. The user experience is like this: Press up a few times then Ctrl+Left a few times. What I see is 1-2 seconds between history changes. At each history change I see the new text appear, then I see cursor ( "_" ) appear at the beginning of the line then slowly progress one character to the right over and over until it reaches the end of the line; then the next line of history appears and this weird animation repeats.  It's as if there are a ton of animations  that must play and complete before the control is returned to the user.  

In other words I press: up , up , up , ctrl + Left.  Then I see
_history 3
h_istory 3
hi_story 3
his_tory 3
hist_ory 3
histo_ry 3
histor_y 3
history_ 3
history _3
history 3_
history 2_
_history 2
h_istory 2
hi_story 2
etc...

**Comment by jjalonso** [comment]
I just open hyper after a long time, and its extremely slow, sometimes writing it does some lag of 1 secs to show the char, something is going on now.

**Comment by megavolt666eblan** [comment]
You write to wrong email, i dont work in hyper

вс, 31 окт. 2021 г., 18:45 Juan Jose A. ***@***.***>:

**Comment by jjalonso** [comment]
Mate, you receiving this email not because i'm writing you an email, you
opened a ticket in Hyper just three days ago and im commenting under, lmao.

**Comment by jjalonso** [comment]
Or you just subscribed to all issues.

**Comment by Schr3da** [comment]
@asdf23 deactivate all custom installed plugins (if any) and try again.
If it still lags - can you please share your device spec?
